code/modules/events/hostile_infestation.dm

```diff
@@ -0,0 +1,125 @@
+//Hostile Infestation: A variant of the infestation event that spawns small numbers of hostile mobs.
+//Currently only fires if there are 2 or more security officers
+
+#define LOC_KITCHEN 0
```

Comment:
Oldcode isn't just a buzzword. Big chunks of it legitimately appear to have been written by lobotomized chimpanzees. Generally, if something in existing code doesn't appear to make sense, it actually doesn't make sense. This is truer the older the code is.